### PR TITLE
chore(docs): update Contributor Guidelines for Hacktoberfest

### PR DESCRIPTION
**Pull request by AlexLakatos**
## Changes proposed in this pull request

- Update contribution guidelines to remove yarn, and add pnpm

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ account.
 Never heard of Interledger before, or you would like to learn more? Here are some good places to start:
 
 - [Good first issues](https://github.com/interledger/rafiki/contribute)
-- [Interledger](https://interledger.org/)
+- [Interledger Explainder Video](https://twitter.com/Interledger/status/1567916000074678272)
```
Comment (wilsonianb) [inline README.md:24]:
```suggestion
- [Interledger Explainer Video](https://twitter.com/Interledger/status/1567916000074678272)
```

Comment (AlexLakatos) [reply README.md]: Nice catch. I ran Grammarly over it before committing, you were better than Grammarly!

File: README.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ account.
 Never heard of Interledger before, or you would like to learn more? Here are some good places to start:
 
 - [Good first issues](https://github.com/interledger/rafiki/contribute)
-- [Interledger](https://interledger.org/)
+- [Interledger Explainer Video](https://twitter.com/Interledger/status/1567916000074678272)
+- [Interledger Website](https://interledger.org/)
 - [Payment pointers](https://paymentpointers.org/)
 - [Web monetization](https://webmonetization.org/)
 - [Coil developers](https://developers.coil.com/)
```
Comment (mkurapov) [inline README.md:28]: L47 in this Readme should be "Local Development Enironment" -> "Local Development Environment"

File: .github/contributing.md [hunk 3]
```diff
@@ -34,74 +34,37 @@ Discussions are where we have conversations about Rafiki.
...
 The team will review your pull request as soon as possible.
 
-## Working in the rafiki repository
-
-This project uses yarn 2. We use [zero-installs](https://yarnpkg.com/features/zero-installs)
-which means you won't need to install dependencies when you clone the repo.
-This does have [security implications](https://yarnpkg.com/features/zero-installs#does-it-have-security-implications)
-that are checked in CI.
+### :books: Documentation
+We maintain documentation for Rafiki on [rafiki.dev](https://github.com/interledger/rafiki.dev). A list of issues being tracked across the Interledger ecosystem (including rafiki) is maintained in the [Documentation project](https://github.com/orgs/interledger/projects/5/views/1).
```
Comment (mkurapov) [inline .github/contributing.md:47]: Should rafiki.dev be included here even if it's still quite fresh?

Comment (AlexLakatos) [reply .github/contributing.md:47]: I'll reword that to be clear it's still fresh.

File: README.md (hunk 2 above)
Comment (AlexLakatos) [reply README.md:28]: Fixed that now.

File: .github/contributing.md [hunk 4]
```diff
@@ -34,74 +34,37 @@ Discussions are where we have conversations about Rafiki.
...
-Additionally, `$INIT_CWD` will always point to the directory running the script.
-
-We utilize this to write shared scripts once:
+## Working in the rafiki repository
 
-```shell
-# Lint in the current workspace
-cd packages/backend
-yarn lint # runs yarn lint:local
+This project uses pnpm. A list of steps for setting up a [local development environment](https://github.com/interledger/rafiki#local-development-enironment) can be found in the Readme.
```
Comment (mkurapov) [inline .github/contributing.md:52]:
```suggestion
This project uses pnpm. A list of steps for setting up a [local development environment](https://github.com/interledger/rafiki#local-development-environment) can be found in the Readme.
```

**Review by mkurapov — APPROVED** [review]

**Pull request merged**